src/Illuminate/Foundation/Testing/Concerns/MocksApplicationServices.php

```diff
@@ -45,6 +45,44 @@ public function expectsEvents($events)
     }
 
     /**
+     * Specify a list of events that should not be fired for the given operation.
+     *
+     * These events will be mocked, so that handlers will not actually be executed.
+     *
+     * @param  array|mixed  $events
```

Comment:
The method above (`expectsEvents`) uses exactly the same PHPDoc - should I change both to `mixed` only?